src/main/res/values/strings.xml

```diff
@@ -447,7 +447,7 @@
     <!-- title shown above a list contacts where one should be selected (eg. when a webxdc attempts to send a message to a chat) -->
     <string name="send_message_to">Send Message to…</string>
     <string name="enable_realtime">Realtime Webxdc Channels</string>
-    <string name="enable_realtime_explain">Enable Webxdc realtime APIs to create direct connections between devices.</string>
+    <string name="enable_realtime_explain">Enable Webxdc realtime APIs to create direct connections between devices. If this is enabled, your chat partners may be able to see your IP address.</string>
```

Comment:
this is a bit shorter - and also removes the advice "enable ... to ...", which is a bit off as it will be enabled by default in the future.

```suggestion
    <string name="enable_realtime_explain">Webxdc realtime APIs allow direct connections between devices. If enabled, chat partners may be able to see your IP address.</string>
```

nb: we're otherwise not talking about much "webxdc" inside Delta Chat, but more about "apps" - but that is a bit forth and back anyways :) so, as being advanced, probably okay to call the option "Realtime Webxdc Channels" - alternative may be "Realtime App Channels", however, i do not really like that and also  do not have a really good other suggestion here

Reply:
> nb: we're otherwise not talking about much "webxdc" inside Delta Chat,

I kept it because the title is also saying "webxdc" but otherwise I am up for renaming it, just saying "apps" is a bit broad/unclear tho